lib/roku-develop.coffee

```diff
@@ -419,19 +423,31 @@ module.exports        = RokuDevelop =
 
     archive.pipe outputStream
 
-    # Compile a list of files and directories to be compressed
-    @addFilesToArchive(archive, @projectDirectory)
+    @getRokuDevIgnores (rokuDevIgnores) =>
+      console.debug(rokuDevIgnores)
+      # Compile a list of files and directories to be compressed
+      @addFilesToArchive archive, @projectDirectory, rokuDevIgnores
+
+      # Finish the compression, calling the output stream's close handler
+      archive.finalize()
 
-    # Finish the compression, calling the output stream's close handler
-    archive.finalize()
     return true
 
-  addFilesToArchive: (archive, parentEntry) ->
+  addFilesToArchive: (archive, parentEntry, rokuDevIgnores) ->
     for entry in parentEntry.getEntriesSync()
       baseName = entry.getBaseName()
       pathname = entry.getRealPathSync()
+      relPath = entry.getPath()
+        .replace(@projectDirectory.getRealPathSync(), '')
```

Comment:
I think this line should be:
`       .replace(@projectDirectory.getPath(), '')`

since the projectDirectory path may contain symlinks.